stable/prometheus/templates/pushgateway-deployment.yaml

```diff
@@ -43,9 +43,9 @@ spec:
           readinessProbe:
```

Comment:
what about the liveness probe?

Reply:
Hmm, I don't know why the liveness probe doesn't exist. Should I add to this PR?

Reply:
Please do.